integration/rpctest/rpc_harness.go

```diff
@@ -482,8 +483,8 @@ func generateListeningAddresses() (string, string) {
 	localhost := "127.0.0.1"
 
 	portString := func(minPort, maxPort int) string {
-		port := minPort + numTestInstances + ((20 * processID) %
-			(maxPort - minPort))
+		rand.Seed(time.Now().UnixNano())
```

Comment:
It's enough to set the seed only once, for example in line 484.

Reply:
yep, makes sense, thanks